docs/sources/alerting/unified-alerting/high-availability/_index.md

```diff
@@ -0,0 +1,25 @@
++++
+title = " About alerting high availability"
+description = "High availability"
+keywords = ["grafana", "alerting", "tutorials", "ha", "high availability"]
+weight = 450
++++
+
+# About alerting high availability
+
+The Grafana alerting system has two main components: a `Scheduler` and an internal `Alertmanager`. The `Scheduler` evaluates your [alert rules]({{< relref "../fundamentals/evaluate-grafana-alerts.md" >}}), while the internal Alertmanager manages **routing** and **grouping**.
+
+When running Grafana alerting in high availability, the operational mode of the scheduler remains unaffected, and each Grafana instance evaluates all alerts. The operational change happens in the Alertmanager when it deduplicates alert notifications across Grafana instances.
+
+{{< figure src="/static/img/docs/alerting/unified/high-availability-ua.png" class="docs-image--no-shadow" max-width= "750px" caption="High availability" >}}
+
+The coordination between Grafana instances happens via [a Gossip protocol](https://en.wikipedia.org/wiki/Gossip_protocol). Alerts are not gossiped between instances and each scheduler delivers the same volume of alerts to each Alertmanager.
+
+The two types of messages gossiped between Grafana instances are:
+
+- Notification logs: Who (which instance) notified what (which alert).
+- Silences: If an alert should fire or not.
+
+These two states are persisted in the database periodically and when Grafana is gracefully shutdown.
+
+For configuration instructions, see [enable alerting high availability]({{< relref "./enable-alerting-ha.md" >}}).
```

Comment:
I've been using "refer to" rather than "see", but we haven't had a discussion about how to specifically word xrefs. So, you're not wrong, just throwing it out there.

Reply:
I prefer "refer to" as well.